src/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "scripts": {
+    "test:cmd": "npx cypress run --browser=chrome",
+    "test:interactive":"npx cypress run --browser=chrome --headed"
+  },
+  "cypress-cucumber-preprocessor": {
+    "nonGlobalStepDefinitions": true
+  },
+  "devDependencies": {
+    "cypress": "^8.7.0",
+    "cypress-cucumber-preprocessor": "^4.3.1"
+  },
+  "dependencies": {
+    "cypress-configuration-builder": "^1.2.4"
```

Comment:
Move to devDeps

Reply:
Fixed it in the latest commit, whenever i save the file the prettier will format the file, removed the video